src/components/Question.js

```diff
@@ -15,9 +15,11 @@ const Question = ({ question, handleAnswerSelect, selectedAnswer }) => {
                 id={`${question.id}.${answer.id}`}
                 name={question.id}
                 value={answer.id}
-                onChange={handleAnswerSelect}
+                onChange={e => handleAnswerSelect(e, question.type)}
                 label={`${answer.text}`}
-                checked={answer.id === selectedAnswer}
+                checked={
+                  selectedAnswer ? selectedAnswer.includes(answer.id) : false
+                }
```

Comment:
```suggestion
                defaultChecked={selectedAnswer.includes(answer.id)}
```
I don't think we need a ternary here. To fix the issue of the React controlled component console error, we can use `defaultChecked` which turns it into an uncontrolled component: https://reactjs.org/docs/uncontrolled-components.html#default-values

We can default `selectedAnswer` to an empty array in the parameters which means we don't have to check whether it's undefined in line 20: `const Question = ({ question, handleAnswerSelect, selectedAnswer = [] }) => {`

Reply:
Done!